### PR TITLE
Add RobotoConf 2024.

### PR DESCRIPTION
**Pull request by johnjohndoe**
- No idea when the CfP started but I guess it is not that important.

### CODE REVIEW

File: _conferences/robotoconf-verona-2024.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+name: "RobotoConf"
+website: http://www.robotoconference.com
```
Comment (cketti) [inline _conferences/robotoconf-verona-2024.md]:
```suggestion
website: https://www.robotoconference.com/
```

**Review by cketti — COMMENTED** [review]
I've not heard of "#pragma mark" before. To make reasonably sure this isn't a scam I've contacted one of the people listed as a speaker to confirm they'll be speaking at the conference. I'll report back once I've got a response.

**Comment by cortinico** [comment]
> I've not heard of "#pragma mark" before

I know them well and can vouch for them 👍 (for context: I know one of the organizers)

**Review by cketti — APPROVED** [review]

**Comment by johnjohndoe** [comment]
\o/

**Pull request merged**